data/json/npcs/tacoma_ranch/NPC_ranch_foreman.json

```diff
@@ -938,7 +938,7 @@
     "followup": "MISSION_RANCH_FOREMAN_17",
     "dialogue": {
       "describe": "We need help…",
-      "offer": "Just flipping through the book, I can tell that one ingredient in most of the alcoholic drinks that we don't have a large supply of is sugar.  What alcohol we have been able to loot isn't going to last us long, so starting our first large batch is a priority.  Could you bring in 80 units of sugar?  That should last us until we can start producing our own supply.",
+      "offer": "Just flipping through the book, I can tell that one ingredient in most of the alcoholic drinks that we don't have a large supply of is sugar.  What alcohol we have been able to loot isn't going to last us long, so starting our first large batch is a priority.  Could you bring in 80 lbs of sugar?  That should last us until we can start producing our own supply.",
```

Comment:
Unfortunately, this will bring an issue like in #74463, e.g. if I'm playing with metric system, I'll still get `80 lbs`. That's not a blocker, but may be a regression.